data.sql

```diff
@@ -9,4 +9,16 @@ values (3,'Pikachu','2021-01-07',15.04,false,1);
 INSERT INTO ANIMALS (id,name,date_of_birth,weight_kg,neutered,escape_attempts)
 values (4,'Devimon','2017-05-12',11.0,true,5);
 
+INSERT INTO ANIMALS (id,name,date_of_birth,weight_kg,neutered,escape_attempts)
+values (5,'Charmander','2020-02-08',11.0,true,0),(6,'Plantmon','2021-11-15',5.7,true,2),
+(7,'Squirtle','1993-04-02',12.13,false,3),(8,'Angemon','2005-06-12',45.0,true,3),
```

Comment:
- [ ] Squirtle's weight should be `-12.3` and not `12.3`
- [ ] Angemon's weight should be `-45.0` and not `45.0`

Reply:
Dropped the table and loaded fresh data accordingly.